src/sentry/runner/commands/devserver.py

```diff
@@ -18,11 +18,22 @@
         "sentry",
         "run",
         "post-process-forwarder",
-        "--entity=all",
+        "--entity=errors",
         "--loglevel=debug",
         "--commit-batch-size=100",
         "--commit-batch-timeout-ms=1000",
     ],
+    "post-process-forwarder-transactions": [
```

Comment:
do we want to include this down below where we do the following:

```python
if eventstream.requires_post_process_forwarder():
    daemons += [_get_daemon("post-process-forwarder")]
```

Reply:
Thanks, added